### PR TITLE
Pin and toast

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,9 +31,14 @@ function App() {
...
+        location.pathname === "/archived" ||
+        location.pathname === "/labels" ||
+        location.pathname === "/trashed" ? (
+        <Sidebar />
+      ) : null}
 
       {/* <Navbar /> */}
-      {/* <Sidebar /> */}
       <Routes>
         {/* <Route path="/mockman" element={<Mockman />} /> */}
```
Comment (MuskanChhatrasal) [inline src/App.js:43]: Remove commented code.

File: src/Components/EditCard/EditCard.jsx [hunk 2]
```diff
@@ -1,122 +1,144 @@
...
-							}}
-						/>
-					</div>
-					<div>
-
-						<span className="pin-icon" onClick={() => {
+            {/* <span className="pin-icon" onClick={() => {
 							setEditCardDetails({
 								...editCardDetails,
 								pinned: !editCardDetails.pinned,
```
Comment (MuskanChhatrasal) [inline src/Components/EditCard/EditCard.jsx:85]: Take care of the alignment. Use prettier

File: src/Components/Header/Navbar.jsx [hunk 3]
```diff
@@ -2,52 +2,53 @@ import React from 'react'
...
 const NavBar = () => {
 	const { logout } = useAuth();
 	return (
-		<header class="header">
-			<Link to="/notes" className='logo'> <i class="fa-solid fa-book-open-reader"></i>  Fresh-Keep</Link>
+		<header className="header">
+			<Link to="/notes" className='logo'> <i className="fa-solid fa-book-open-reader"></i>  Fresh-Keep</Link>
 
-			{/* <form action="" class="search-form">
+			{/* <form action="" className="search-form">
```
Comment (MuskanChhatrasal) [inline src/Components/Header/Navbar.jsx]: Here also remove the commented code.

File: src/Pages/LabelPage/LabelPage.jsx [hunk 4]
```diff
@@ -1,175 +1,207 @@
...
+    })();
+  }, [labels]);
+
+  return (
+    <>
+      <div className="main-container">
+        <div className="filter-container-landing">
+          <h1 className="filter-heading">Tags</h1>
+          <span className="filter-sub-heading">
+            <label>
```
Comment (MuskanChhatrasal) [inline src/Pages/LabelPage/LabelPage.jsx:58]: Remove unnecessary spaces.

**Review by MuskanChhatrasal — APPROVED** [review]
Just suggested few things implement those. Else everything looks good to merge

File: src/Pages/NotesPage/NotesPage.jsx [hunk 5]
```diff
@@ -1,134 +1,173 @@
...
+              <span>High to Low</span>
+            </label>
+
+            <br />
+            <label>
+              <input
+                type="radio"
+                name="group1"
+                onClick={() =>
+                  setLabels({ ...labels, HighToLow: false, LowToHigh: true })
```
Comment (Dev-AkshatGupta) [inline src/Pages/NotesPage/NotesPage.jsx:92]: Suggestion try to make them controlled .

Comment (aayushjain-code) [reply src/Pages/NotesPage/NotesPage.jsx:92]: sure thanks

File: src/Pages/NotesPage/NotesPage.jsx [hunk 6]
```diff
@@ -1,134 +1,173 @@
...
+              />
+              <span>Newest</span>
+            </label>
+          </span>
+          {/* <button
+            className="button primary"
+            onClick={() => setShowNotes(!showNotes)}
+          >
+            Show {showNotes ? `All ` : `Only Pinned `}Notes
+          </button> */}
```
Comment (Dev-AkshatGupta) [inline src/Pages/NotesPage/NotesPage.jsx:128]: remove unnecessary code

**Review by Dev-AkshatGupta — APPROVED** [review]
Else everything looks good to me you can go ahead anad merge it

**Pull request merged**